.github/workflows/create-cluster.yml

```diff
@@ -0,0 +1,71 @@
+name: Create Test Cluster
+on:
+  workflow_call:
+    inputs:
+      flavor:
+        description: Flavor
+        type: string
+        required: true
+      name:
+        description: Cluster name
+        type: string
+        required: true
+      lifespan:
+        description: Cluster lifespan
+        type: string
+        default: 48h
+        required: false
+      args:
+        description: Comma separated flavor arguments. Ex. nodes=5,main-image=main:tag
+        type: string
+        required: false
+        default: ""
+      wait:
+        description: Whether to wait for the cluster readiness
+        type: boolean
+        required: false
+        default: false
+jobs:
+  infra:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download infractl
+        run: |
+          curl --fail -sL https://infra.rox.systems/v1/cli/linux/amd64/upgrade \
```

Comment:
How about adding `-S`?

>-S, --show-error
 >      When used with -s it makes curl show an error message if it fails.